packages/smithy-http/.changes/next-release/smithy-http-bugfix-20251023131033.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "bugfix",
+  "description": "Fix `InvalidSignatureException` caused by mismatched list header formatting during signing (e.g., `MyHeader: value1, value2` vs `MyHeader: value1,value2`)."
```

Comment:
```suggestion
  "description": "Standardized header concatenation delimiter to "," to avoid signature mismatches (e.g., `MyHeader: value1, value2` vs `MyHeader: value1,value2`)."
```